app/javascript/dashboard/components/widgets/conversation/ConversationHeader.vue

```diff
@@ -133,7 +112,7 @@ export default {
 
 <template>
   <div
-    class="flex flex-col items-center justify-between px-4 py-2 border-b bg-n-background border-n-weak md:flex-row"
+    class="flex flex-col items-center justify-between px-3 py-2 border-b bg-n-background border-n-weak md:flex-row h-12"
```

Comment:
@pranavrajs Adding fixed height will break the header in smaller screen.
<img width="471" alt="image" src="https://github.com/user-attachments/assets/fd8ec441-49af-42e1-a271-33c4cd6b1639" />